packages/erc20z/README.md

```diff
@@ -2,7 +2,7 @@
 
 The Zora Sale Timed Sale Strategy introduces a new fixed price and unlocks secondary sales on Uniswap V3. New tokens minted will have a fixed price of 0.000111 ETH (111 sparks).
 
-Upon calling setSale() a sale will be created for the 1155 NFT provided. In this function, it will also create an ERC20z token and a Uniswap V3 Pool. The ERC20z token will be used as a pool pair (WETH / ERC20z) as well as enable wrapping and unwrapping tokens from 1155 to ERC20z and vice versa.
+Upon calling setSale() a sale will be created for the 1155 NFT provided. In this function, it will also create an ERC20z token and an Uniswap V3 Pool. The ERC20z token will be used as a pool pair (WETH / ERC20z) as well as enable wrapping and unwrapping tokens from 1155 to ERC20z and vice versa.
```

Comment:
this isn't correct - we use "an" before words that begin with a vowel sound, and "a" before words that begin with a consonant sound.
Even though "Uniswap" starts with the letter 'U', it's pronounced with a 'you' sound (like "YOU-niswap"), which is a consonant sound. Therefore, we use "a" instead of "an".